lib/hello/accounts.ex

```diff
@@ -0,0 +1,86 @@
+defmodule Hello.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+
+  import Ecto.Query, warn: false
```

Comment:
@JuanMaGiraldo Why is this import required?

Reply:
It's required to use the `Ecto.Query` functions like: from, select, where...